File: fizzbuzz.php
<?php
declare(strict_types=1);
 
for ($i = 1; $i <= 100; $i++) {
    if ($i % 15 === 0) {
        echo 'FizzBuzz';
    } elseif ($i % 3 === 0) {
        echo 'Fizz';
    } elseif ($i % 5 === 0) {
        echo 'Buzz';
    } else {
        echo $i;
    }
    echo PHP_EOL;
}

File: fizzbuzz.go
package main
import ("fmt")
func main() {
    i := 1
    for i < 101 {
        switch {
        case i%15 == 0:
            fmt.Println("FIZZ BUZZ!")
        case i%3 == 0:
            fmt.Println("FIZZ!")
        case i%5 == 0:
            fmt.Println("BUZZ!")
        default:
            fmt.Println(i)
        }
        i++
    }
}
